packages/core/test/invoke.test.ts

```diff
@@ -3121,7 +3122,7 @@ describe('invoke', () => {
       on: {
         '*': {
           actions: ({ event }) => {
-            actual.push(event.type);
+            actual.push(event as any);
```

Comment:
we shouldn't have to cast here, what is the reporter error without it?

Reply:
This test machine does not declare its event types, so any event it handles will be of type `AnyEventObject`--which is not `DoneActorEvent<void>` (which `actual` is an array of).  We could cast to `DoneActorEvent<void>`, but that seemed a little wordy. 🙂

Reply:
Previously, `actual` was `string[]`, which did not need a type assertion.